layers/auto-completion/packages.el

```diff
@@ -263,18 +264,6 @@
                                                                eshell-mode-hook)))
     :config
     (progn
```

Comment:
Oh you are right, if it is a one liner then it is indeed unnecessary. In this case I usually but the keyword and the argument on the same line.